GameData/RealismOverhaul/RO_SuggestedMods/RealScaleBoosters/RO_RSB_Delta.cfg

```diff
@@ -714,15 +697,45 @@
         }
     }
 
-    !MODULE[ModuleSPU]{}
+    !MODULE[ModuleSPU*],*{}
 
-    !MODULE[ModuleRTAntenna*]{}
+    !MODULE[ModuleRTAntenna*],*{}
 
     !RESOURCE,*{}
+
+    //  Avionics batteries 1.4 kWh.
+    //  Can support the DCSS for flights up to 7 hours in duration.
+    //  Assumes that the electricity consumption of the DCSS avionics is 200 W.
+
+    RESOURCE
```

Comment:
Yeah, good point. That seems like probably the best current way to deal with that issue then. Being able to contain 2 tank types would be a good future feature for ModuleFuelTanks, but is definitely not currently possible.